apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1094,6 +1082,7 @@ namespace MWMechanics
         }
 
         // Make surrounding actors within alarm distance respond to the crime
+        // TODO: Vanilla doesn't make neighbors attack player if one of his follower/summon attacks first (probably vanilla bug)
```

Comment:
I don't think it matters that much, because normally your follower won't start combat with a non-aggressive actor anyway. They just respond to ones that are aggressive to you or them. So this would only matter if there is a script using "startcombat" on the follower, or if the player is using console commands. I suppose the best thing would be to follow vanilla, just in case there is a mod that uses startcombat to make the player follower attack and expects the vanilla behavior, but it seems like a very rare case to me.